pkg/scaffold/project/kustomize.go

```diff
@@ -74,10 +74,26 @@ resources:
 - ../rbac/rbac_role.yaml
 - ../rbac/rbac_role_binding.yaml
 - ../manager/manager.yaml
+# Comment the following 3 lines if you want to disable
+# kube-rbac-proxy (https://github.com/brancz/kube-rbac-proxy)
+# which protects your /metrics endpoint.
+- ../rbac/kube-rbac-proxy-service.yaml
```

Comment:
Can we name this file `auth_proxy_service.yaml` to keep it consistent with the roles file (also note the underscore ?